rbx_binary/src/deserializer/state.rs

```diff
@@ -155,7 +155,7 @@ fn find_canonical_property<'de>(
             );
 
             Some(CanonicalProperty {
-                name: canonical_name,
+                name: canonical_name.clone().into(),
```

Comment:
```suggestion
                name: canonical_name.as_ref().into(),
```
This saves us a potential allocation if this is an owned `Cow` since we can just get it as a `&str`.